views/github-create-branch.hbs

```diff
@@ -30,26 +30,28 @@
   <div class="headerImage">
     <img src="/public/assets/jira-and-github.png" alt="Jira and GitHub logos" />
   </div>
+
   <div class="gitHubCreateBranch__title">
     <div class="gitHubCreateBranch__header">Create GitHub Branch</div>
-    <div class="gitHubCreateBranch__subHeader">
-      Creating a branch for <b>{{issue.key}}</b>
-    </div>
+    <div class="gitHubCreateBranch__subHeader skeleton skeleton-text">Creating a branch for <b>{{issue.key}}</b></div>
   </div>
 
-  <form id="createBranchForm" class="aui top-label">
+  <form id="createBranchForm" class="aui top-label createBranchForm">
     <!--TODO: Add condition to display this only for GHES -->
     <div class="field-group">
-      <aui-label for="server">GitHub Enterprise Server instance</aui-label>
-      <select id="ghServers" name="server" class="full-width-field" placeholder="Select a repository">
-        {{#each servers}}
-          <option value="{{id}}">{{server}} / {{appName}}</option>
-        {{/each}}
-      </select>
+      <aui-label for="server" class="skeleton skeleton-text">GitHub Enterprise Server instance</aui-label>
+       <div class="skeleton skeleton-input"></div>
```

Comment:
For input elements, we need to handle it a little differently because the AUI styling gets in the way and I didn't want to end up with !important all over the place. Easier solution here is to simply render a separate skeleton element temporarily in place of the real element.